net-misc/owncloud-client/owncloud-client-2.10.0.6519.ebuild

```diff
@@ -9,13 +9,15 @@ MY_PN="ownCloud"
 DESCRIPTION="Synchronize files from ownCloud Server with your computer"
 HOMEPAGE="https://owncloud.org/"
 SRC_URI="https://download.owncloud.com/desktop/${MY_PN}/stable/${PV}/source/${MY_PN}-${PV}.tar.xz"
+S=${WORKDIR}/${MY_PN}-${PV}
```

Comment:
Moving ${S} around is not related to that change (I tend to keep it close to src_prepare() ) but since nextcloud-client similar change is already merged, let's keep the same here

Reply:
`S` is moved below `SRC_URI` recently since most of the time they are related.